skyportal/app_server.py

```diff
@@ -27,6 +27,7 @@
     ObservingRunHandler,
     PhotometryHandler,
     BulkDeletePhotometryHandler,
+    ObjShortHandler,
```

Comment:
I'd opt for making this `ObjHandler` and implementing `HEAD` here

Reply:
I rescind this comment - I think what you have here is the right way to go

HEAD should just return the HTTP headers that would be returned by GET and no body, so I dont think its the right method for this...

https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/HEAD